packages/cli/create-medusa-app/src/utils/prepare-project.ts

```diff
@@ -138,7 +138,7 @@ export default async ({
     await processManager.runProcess({
       process: async () => {
         const proc = await execute(
-          ["npx medusa migrations run", npxOptions],
+          ["npx medusa migrations run && npx medusa links sync", npxOptions],
```

Comment:
Q: is there a reason why there is many if with the exact same condition? :D also, should we display a new box saying that we are syncing the links?

Reply:
> is there a reason why there is many if with the exact same condition? :D

We had other conditions related to seeding in the condition after it, but then we removed them as it no longer applied with the onboarding, so the conditions became the same now 😄 I'll open another PR to refactor.

> should we display a new box saying that we are syncing the links?

I'm not sure that's necessary really, but I'll look into it as part of the refactor PR 👍🏻

Reply:
should we make it concurrent @shahednasser ?